ingest/api/model.py

```diff
@@ -339,6 +331,23 @@ def validate_wigos_id(self) -> Properties:
 
         return self
 
+    @model_validator(mode="after")
+    def transform_period_to_seconds(self):
+        try:
+            duration = isodate.parse_duration(self.period)
+        except isodate.duration.ParsingError:
```

Comment:
`isodate` does not throw a `ParsingError` if it fails to parse a duration, it throws a `ISO8601Error`. Currently this will lead to `Internal Server Error` if somebody tries to ingest for example:
```
properties: {
   ...
  "level": "2.0", 
  "period": "xd", 
  "function": "mean",
  ...
```

Reply:
Fixed.